src/app/header/header.component.html

```diff
@@ -1,74 +1,157 @@
-<nav class="navbar navbar-default navbar-pf" role="navigation" *ngIf="loggedIn">
-    <div class="navbar-header"></div>
+<nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="loggedIn">
```

Comment:
Not sure what you are doing here, we now have this menu twice in the html - here, and at the bottom of the file.